src/main/resources/templates/casUnsupportedInstitutionLoginView.html

```diff
@@ -0,0 +1,107 @@
+                        <span class="mdc-button__label" th:utext="#{screen.instnloginna.existing.button}"></span>
+                    </a>
+                </section>
+
+                <section id="osfUnsupportedInstitutionLogin">
+                    <div class="card-message">
+                        <p th:utext="#{screen.instnloginna.existing.setpw.message}"></p>
+                    </div>
+                </section>
+
+                <div class="form-wrapper">
+                    <form method="post" id="fm1" th:object="${credential}" th:action="@{${osfUrl.forgotPasswordInstitution}}">
+                        <section class="cas-field cas-field-primary my-3">
+                            <div class="mdc-text-field  d-flex">
+                                <input class="mdc-text-field__input"
+                                       id="username"
+                                       size="256"
+                                       type="email"
+                                       name="forgot_password-email"
+                                       th:accesskey="#{screen.instnloginna.existing.setpw.label.email.accesskey}"
+                                       autocomplete="off" />
+                                <label for="username" class="mdc-floating-label" th:utext="#{screen.instnloginna.existing.setpw.label.email}"></label>
+                            </div>
+                        </section>
+                        <section>
+                            <div th:replace="fragments/submitbutton :: submitButton (buttonDisabled=false, buttonCustomization='button-osf-blue', messageKey='screen.instnloginna.existing.setpw.button')" />
+                        </section>
+                    </form>
+                </div>
+
+                <hr class="my-4" />
+                <section class="text-with-mdi">
+                    <i class="mdi mdi-account-question-outline mdi-before-text"></i>
+                    <span>
+                            <a href="https://help.osf.io/hc/en-us/articles/360019737194-Sign-in-to-OSF" th:utext="#{screen.generic.link.support}"></a>
+                        </span>
+                </section>
+
+            </section>
+        </div>
+
+        <script type="text/javascript" th:inline="javascript">
+            /*<![CDATA[*/
+                var i = /*[[@{#{screen.generic.button.wip}}]]*/ 'One moment please...' ;
+                var j = /*[[@{#{screen.instnloginna.existing.setpw.button}}]]*/ 'Set a password' ;
+            /*]]>*/
+            $(window).on('pageshow', function () {
+                $(':submit').prop('disabled', false);
+                $(':submit').attr('value', j);
+            });
+            $(document).ready(function () {
+                $("#fm1").submit(function () {
+                    $(":submit").attr("disabled", true);
+                    $(":submit").attr("value", i);
+                    return true;
+                });
+            });
+        </script>
```

Comment:
Since you added a new form submission, here is the scripts (default) for pre- and post- processing the submit button.